src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Handlers/HandlerRequestType.cs

```diff
@@ -13,6 +13,6 @@ internal enum HandlerRequestType
         /// <summary>
         /// Request type for connection request.
```

Comment:
Modify the comment to match the variable name

Something on the lines of `Context for Connection Request`